tests/test_examples.py

```diff
@@ -515,7 +515,7 @@ def test_NAE_QSC_solve():
 
     # this has all the constraints we need,
     #  iota=False specifies we want to fix current instead of iota
-    cs = get_NAE_constraints(eq, qsc, iota=False, order=1, N=eq.N)
+    cs = get_NAE_constraints(eq, qsc, iota=False, order=1, fix_lambda=True, N=eq.N)
```

Comment:
as an additional test, can we try solving with and without fixed lambda and make sure they give the same lambda near axis?